blob/s3blob/s3blob.go

```diff
@@ -188,10 +189,23 @@ func (b *bucket) NewRangeReader(ctx context.Context, key string, offset, length
 		}
 		return nil, err
 	}
+	// Default size to ContentLength, but that's incorrect for partial-length reads,
```

Comment:
How about wrapping this inside a function?

Reply:
Done.